haystack/modeling/training/base.py

```diff
@@ -156,8 +149,8 @@ def __init__(
         :param lr_schedule: An optional scheduler object that can regulate the learning rate of the optimizer
         :param evaluate_every: Perform dev set evaluation after this many steps of training.
         :param eval_report: If evaluate_every is not 0, specifies if an eval report should be generated when evaluating
-        :param use_amp: Whether to use automatic mixed precision with Apex. One of the optimization levels must be chosen.
-                        "O1" is recommended in almost all cases.
+        :param use_amp: Whether to use automatic mixed precision (AMP) natively implemented in PyTorch.
+                        Find more information at https://pytorch.org/docs/stable/amp.html
```

Comment:
let's add the link to our own documentation, in particular the optimization page, as well.